apps/android/app/src/main/java/chat/simplex/app/views/helpers/CustomTimePicker.kt

```diff
@@ -123,8 +123,8 @@ data class TimeUnitLimits(
         CustomTimeUnit.Minute -> TimeUnitLimits(CustomTimeUnit.Minute, maxValue = 120)
         CustomTimeUnit.Hour -> TimeUnitLimits(CustomTimeUnit.Hour, maxValue = 72)
         CustomTimeUnit.Day -> TimeUnitLimits(CustomTimeUnit.Day, maxValue = 60)
-        CustomTimeUnit.Week -> TimeUnitLimits(CustomTimeUnit.Week, maxValue = 12) // TODO in 5.2 - 54
-        CustomTimeUnit.Month -> TimeUnitLimits(CustomTimeUnit.Month, maxValue = 3) // TODO in 5.2 - 12
+        CustomTimeUnit.Week -> TimeUnitLimits(CustomTimeUnit.Week, maxValue = 54)
```

Comment:
why not 52? I think it's traditional for 1 year, 54 is a strange number a bit...